rmgpy/molecule/molecule.py

```diff
@@ -2759,6 +2759,16 @@ def get_surface_sites(self):
         cython.declare(atom=Atom)
         return [atom for atom in self.atoms if atom.is_surface_site()]
 
+    def is_bidentate(self):
+        """
+        Return ``True`` if the adsorbate contains at least two binding sites,
+        or ``False`` otherwise.
+        """
+        cython.declare(atom=Atom)
+        if len([atom for atom in self.atoms if atom.is_surface_site()])>=2:
```

Comment:
we also return `True` if the molecule is tridentate. Would we like to reflect that in the docstring (if not in the function name)?

Reply:
Good catch. I renamed the function to is_multidentate since all multidentate species can have resonance.